py/torch_tensorrt/ts/ts_input.py

```diff
@@ -0,0 +1,108 @@
+from enum import Enum
+from typing import List, Dict, Any, Tuple, Optional
+
+import torch
+
+from torch_tensorrt import _C
+from torch_tensorrt import _enums
+from torch_tensorrt import _Input
+from torch_tensorrt._Input import Input
+
+
+class TSInput(Input):
```

Comment:
This should be a hidden class, we dont want people using this, handle any conversion to TSInput internally

Reply:
Yeah that's the intended usage. In the current design, for torchscript, they still use the same interface 
`torch_tensorrt.Input()` and we internally convert into `TSInput` in the `ts/_compile_spec.py` file. Anything different that you have in mind ?

Reply:
No this would be fine